### DISCUSSION TITLE
How to match github.com's syntax highlighting in GitHub Pages?

### DISCUSSION THREAD

**Discussion by pchemguy** [github-linguist/linguist#6006] [General]
How to match GitHub.com's syntax highlighting in GitHub Pages?

**Comment by lildude** [comment]
You can use [Rouge](https://github.com/rouge-ruby/rouge) which has a [GitHub theme](https://github.com/rouge-ruby/rouge/blob/master/lib/rouge/themes/github.rb), which is not 💯 identical, but it is quite close.

**Reply by pchemguy** [reply]
GitHub is on the left. Not quite close.

![Clipboard-1](https://user-images.githubusercontent.com/39730837/183648284-4afb77d3-e0ab-43f2-93a9-5edd463bbe3b.jpg)

**Reply by lildude** [reply]
🤔 I think GitHub is actually on the left and I think this might be your theme lacking the right CSS.

You might need to generate the CSS file as detailed in [GitHub's docs](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll#syntax-highlighting) and include it in your theme.

If you're still getting the wrong syntax highlighting, you'll need to report it to the Rouge maintainers.

**Reply by pchemguy** [reply]
Yes, on the left.
